admin/class-create-block-theme-admin.php

```diff
@@ -598,6 +654,69 @@ function build_child_style_css( $theme ) {
 */";
 	}
 
+	function build_blank_functions( $theme ) {
+		$slug = $theme['slug'];
+		$name = $theme['name'];
+	
+	return "<?php
+/**
+ * {$name} functions and definitions
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
+ *
+ * @package {$name} 
+ * @since {$name} 1.0
+ */
+
+if ( ! function_exists( '{$slug}_support' ) ) :
+
+	/**
+	 * Sets up theme defaults and registers support for various WordPress features.
+	 *
+	 * @since {$name} 1.0
+	 *
+	 * @return void
+	 */
+	function {$slug}_support() {
+
+		// Enqueue editor styles.
+		add_editor_style( 'style.css' );
+
+	}
+
+endif;
+
+add_action( 'after_setup_theme', '{$slug}_support' );
+
+if ( ! function_exists( '{$slug}_styles' ) ) :
+
+	/**
+	 * Enqueue styles.
+	 *
+	 * @since {$name} 1.0
+	 *
+	 * @return void
+	 */
+	function {$slug}_styles() {
+
+		// Register theme stylesheet.
+		wp_register_style(
+			'{$slug}-style',
+			get_template_directory_uri() . '/style.css',
+			array(),
+			wp_get_theme()->get( 'Version' )
+		);
+
+		// Enqueue theme stylesheet.
+		wp_enqueue_style( '{$slug}-style' );
+
+	}
+
+endif;
+
+add_action( 'wp_enqueue_scripts', '{$slug}_styles' );";
```

Comment:
Maybe we shouldn't include this and instead encourage theme developers to use the block styles as much as possible?

Reply:
yeah, can we remove functions.php entirely?